api.md

```diff
@@ -280,27 +280,45 @@ with [`add_data_node`](#add_data_node) first before attaching it.
 ---
 ## add_data_node() [](add_data_node)
 
-Add a new data node to the database to be used for creating
-distributed hypertables.
-
-Newly created distributed hypertables will be able to make use of this
+Add a new data node to the database to be used by distributed
+hypertables. This function needs to be executed on the access node by
+a superuser or a user with `USAGE` privileges on the `timescaledb_fdw`
+foreign data wrapper (superuser privileges might still be required on
+the data node). It is assumed that the user role adding a data node
+already has a way to authenticate with the data node being
+added. Alternatively, a password can be provided on the command line
+if the data node accepts password-based authentication and no password
+is provided in a local password file on the access node. Please refer
+to [Setting up Multi-Node TimescaleDB][multinode] for more information
+about node-to-node authentication.
```

Comment:
After reviewing this, this feels like maybe too much in the weeds for the API docs?  Maybe something simpler like:

```suggestion
Add a new data node to the database to be used by distributed
hypertables.   For security purposes, the data node validates the connection
attempt from the access node; TimescaleDB currently supports several 
different such authentication methods for flexibility (including trust, user
mappings, password, and certificate methods).  Please refer
to [Setting up Multi-Node TimescaleDB][multinode] for more information
about node-to-node authentication.
```

Reply:
I think permissions/superuser requirements should be part of reference docs. This will probably trip people up otherwise. Don't mind shortening in general though.